core-services/egov-enc-service/src/main/resources/db/migration/main/V20230930120401__eg_enc.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS  eg_enc_mdms_config;
+
+CREATE TABLE public."eg_enc_mdms_config"
+(
+    id SERIAL,
+    tenant_id text NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE UNIQUE INDEX eg_enc_mdms_config_tenant_id ON eg_enc_mdms_config (tenant_id);
```

Comment:
Why is this index required when there is no where clause?

Reply:
This is added to future proof. If we have tenant level configurations as well in this table, having hte index will help.